src/containers/Trigger/Trigger.tsx

```diff
@@ -31,11 +31,17 @@ const triggerFrequency = [
 ];
 
 const FormSchema = Yup.object().shape({
-  flowId: Yup.object().nullable().required('Flow is required'),
+  flowId: Yup.object().required('Flow is required'),
```

Comment:
- [ ] Please check this

![Screenshot from 2021-03-16 17-03-20](https://user-images.githubusercontent.com/26589967/111302668-b32a7100-8679-11eb-97d5-e7d5ee65d140.png)

- [ ] If we submit an empty form it's not showing validation.